config/ci_configs/amip_edonly_bucket.yml

```diff
@@ -1,12 +1,9 @@
 FLOAT_TYPE: "Float32"
 albedo_model: "CouplerAlbedo"
 atmos_config_file: "config/longrun_configs/amip_target_edonly.yml"
-checkpoint_dt: "720hours"
```

Comment:
This is the default value now (also we don't need to checkpoint regular CI runs)